### PR TITLE
monogamous SETH, L2 config improvements

### PR DESCRIPTION
**Pull request by d10r**
* remove WETH support from SETH because it's error prone and we can't update if a network's WETH contract changes
* improve config of new L2s
* add L2 deployment addresses

### CODE REVIEW

File: packages/js-sdk/src/getConfig.js [hunk 1]
```diff
@@ -125,6 +125,13 @@ Superfluid_getConfig = module.exports = function getConfig(chainId, version) {
         10: {
             // op mainnet
             nativeTokenSymbol: "ETH",
+            resolverAddress: "0x743B5f46BC86caF41bE4956d9275721E0531B186",
+            versions: {
+                v1: {
+                    subgraphQueryEndpoint:
+                        "https://api.thegraph.com/subgraphs/name/superfluid-finance/protocol-v1-optimism-mainnet",
```
Comment (0xdavinchee) [inline packages/js-sdk/src/getConfig.js]: optimism-mainnet => optimism-main

**Comment by hellwolf** [comment]
I will look into this too

File: packages/ethereum-contracts/contracts/tokens/SETH.sol [hunk 2]
```diff
@@ -8,61 +8,19 @@ import {
...
-        } // else it is weth, it is already implemented in upgrade (ByWETH)
-    }
-}
 
 /**
  * @dev Super ETH (SETH) custom super token implementation
  * @author Superfluid
  */
 contract SETHProxy is ISETHCustom, CustomSuperTokenBase, UUPSProxy {
+    event TokenUpgraded(address indexed account, uint256 amount);
```
Comment (hellwolf) [inline packages/ethereum-contracts/contracts/tokens/SETH.sol:17]: why these need to be defined? is it not part of ISETHCustom(ISuperToken) ?

**Review by hellwolf — CHANGES_REQUESTED** [review]
just one small question.

**Review by hellwolf — APPROVED** [review]

**Pull request merged**